sanctuarycomputer/nunu
# Product add to cart

### Description

This PR adds the initial Add to Cart logic:
- Uses Cart.js ajax api to add to cart from product pages
- Updates Cart count on nav when items are added
- Updates added to cart button (contextually) when items are added

### Type(s) of changes


### Motivation for PR
In order to create a more modern e-commerce experience, we use Shopify's AJAX api to add items to the cart, and keep the user on the same page, dynamically updating any content that needs updating.

### How Has This Been Tested?

https://8hdzv69x4jgs9dc7-52674822324.shopifypreview.com

on the link above, you should be able:
- View a product page
- Add product to cart
- Cart count should update
- Add to cart button should update

### Applicable screenshots:



### Follow-up PR

IN a follow up PR, we will want to update the cart UI to display items that were recently added, along with any other necessary data e.g. item totals, price totals, button states, etc.
